gc/base/standard/Scavenger.cpp

```diff
@@ -1677,6 +1677,18 @@ MM_Scavenger::copy(MM_EnvironmentStandard *env, MM_ForwardedHeader* forwardedHea
 	return destinationObjectPtr;
 }
 
+MMINLINE void
+MM_Scavenger::updateForwardedObject(MM_ForwardedHeader* forwardedHeader, omrobjectptr_t destinationObjectPtr, uintptr_t objectAge) {
```

Comment:
while 'update' is very reasonable word, let's align terminology and call it fixup